### PR TITLE
Yup

### PR DESCRIPTION
**Pull request by Torfun**
(You can replace all of this text with your description.)

Before creating your pull request, please check your content against these quality criteria:

- Did you consider search engine optimization (SEO) when you chose the title in the metadata section and the H1 heading (i.e. the displayed title that starts with a single #)?
- For new articles, did you add it to the table of contents?
- Did you update the "ms.date" metadata for new or significantly updated articles?
- Are technical terms and concepts introduced and explained, and are acronyms spelled out on first mention?
- Should this page be linked to from other pages or Microsoft web sites?

For more information about creating content for docs.microsoft.com, see the contributor guide at https://docs.microsoft.com/contribute/.

### CODE REVIEW

**Comment by ktoliver** [comment]
@Yup - Thank you for your contribution. Would you take a moment to sign the CLA? After the CLA is signed, someone will review your changes. Thanks!

Also, could you clarify why you are attempting to merge to repo_sync_working_branch instead of to master?

cc @gewarren

File: mac/ide-tour.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ ms.custom: video
...
+Visual Studio for Mac is a .NET _integrated development environment_ on the Mac that can be used to edit, debug, and build the code and then publish an app. In addition to expected features, such as a standard editor and debugger, Visual Studio for Mac includes compilers, code completion tools, graphical designers, and source control to ease the software development process.
 
-Visual Studio for Mac supports many of the same file types as its Windows counterpart, such as `.csproj`, `.fsproj`, or `.sln` files, and supports features such as EditorConfig, meaning that you can use the IDE that works best for you.
+Visual Studio for Mac supports many of the same file types as its Windows counterparts, such as `.csproj`, `.fsproj`, or `.sln` files, and supports features such as EditorConfig, meaning that you can use the IDE that works best for you.
```
Comment (gewarren) [inline mac/ide-tour.md]: This should be singular - it's referring to Visual Studio on Windows.

**Comment by gewarren** [comment]
@ktoliver I think you meant to tag @Torfun instead of `@yup`.

**Comment by gewarren** [comment]
I'm not sure what's going on here but these changes were already  merged into master so I'm closing this.

**Pull request closed without merging**